### PR TITLE
Created mysql_python_connector file

### PR DESCRIPTION
**Pull request by gr8rithic**

### CODE REVIEW

File: mysql_python_connector.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+mycursor = mydb.cursor()
+
+site_name = input("Enter the name of the site")
+username = input("Enter the username of the site")
+password = input("Enter the password of the site")
+
+sql = "INSERT INTO user_details(site_name,username,password) VALUES(%s, %s, %s)"
+values = (site_name, username, password)
+mycursor.execute(sql,values)
+
```
Comment (mursalfk) [inline mysql_python_connector.py:18]: Please add proper comments to the code.

**Review by mursalfk — CHANGES_REQUESTED** [review]

**Comment by gr8rithic** [comment]
Done the changes and added comments to the program

File: mysql_python_connector.py (hunk 1 above)
Comment (mursalfk) [reply mysql_python_connector.py:23]: Looks cool. Thanks.

Comment (gr8rithic) [reply mysql_python_connector.py:23]: I totally agree with your point. Thanks

**Pull request merged**